### PR TITLE
Registrations in Search Now Have Dates

### LINKED ISSUE #3615: in search results, registrations of the same project cannot be distinguished from one another
from the community repo: https://github.com/CenterForOpenScience/community/issues/276

## Steps
1. Search the OSF for a project that has been registered multiple times (try searching "local versus global")

## Outcome
1. Registrations of the same project are listed as separate results
2. The registrations are indistinguishable from one another
3. They do not necessarily appear next to each other or in recent to oldest order.
screen shot 2015-07-15 at 10 54 27 am

## Suggested Behavior
1. Registrations should be distinguishable from one another.
2. They do not have to be listed as separate results. One suggestion is to collapse registrations of the same project that are returned in search results into one result, with the option to expand that result. On expansion, the different registrations and their dates could be shown and linked to. If this model is followed, I would suggest making the link visible when unexpanded (the title of the registration int he image below) link to the project's registrations page, so that the user is brought to a place where they can see each registration listed.

screen shot 2015-07-15 at 11 06 59 am

Of course, this could almost certainly be improved visually, but the basic idea is there.

### PR DESCRIPTION
**Pull request by Johnetordoff**
<h1> Purpose </h1>
Allows users to distinguish registrations with the same name. Close #3615.
<h1> Changes </h1>
small js changes to format dates and html to display them.
<h1> Side Effects </h1>
None.

### CODE REVIEW

**Comment by sloria** [comment]
Please follow our guidelines for writing good commit messages: https://cosdev.readthedocs.org/en/latest/style_guides/git.html#git

**Comment by sloria** [comment]
Would you mind posting "before" and "after" screenshots to make it easier for others to review?

**Comment by Johnetordoff** [comment]
<h1> Before </h1>
screen shot 2015-08-03 at 12 14 57 pm
<h1> After </h1>
screen shot 2015-08-03 at 12 09 14 pm

File: website/static/js/search.js [hunk 1]
```diff
@@ -244,6 +244,9 @@ var ViewModel = function(params) {
 
                     self.results.push(result);
                 }
+                if(result.category === 'registration'){
+                    result.registered_date = new $osf.FormattableDate(result.registered_date);
```
Comment (sloria) [inline website/static/js/search.js]: As per our style guidelines, use `lowerCamelCase` for variable and attribute names.

**Comment by sloria** [comment]
Pass finished.

**Pull request merged**